validate/format.go

```diff
@@ -0,0 +1,24 @@
+package validate
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/cryptopay-dev/yaga/errors"
+	"gopkg.in/go-playground/validator.v9"
+)
+
+// CheckErrors of validator and return formatted errors:
+func CheckErrors(valErr error) (ok bool, err error) {
+	var fieldsErr validator.ValidationErrors
+	if fieldsErr, ok = valErr.(validator.ValidationErrors); ok {
+		var fields []string
+		for _, field := range fieldsErr {
+			fields = append(fields, strings.ToLower(
+				field.Field(),
+			))
```

Comment:
```
fields := make([]string, len(fieldsErr))
i := 0 
for _, field := range fieldsErr {
	fields[i] = strings.ToLower(field.Field()))
...
```
append an expensive and meaningless operation when we know in advance the size of the slice